public/js/wysiwyg/attachTinyMCE.js

```diff
@@ -0,0 +1,40 @@
+function attachTinyMCE(elementID, siteLocale, mode, image) {
```

Comment:
I would create one wrapper JS file. Which will attach requested editor.
Like editor factory.

And as we using elixir, you can use the power of `require()` in JS for this :)